packages/server/modules/blobstorage/repositories/index.ts

```diff
@@ -43,3 +50,26 @@ export const getAllStreamBlobIdsFactory =
     const res = await tables.blobStorage(deps.db).where({ streamId }).select('id')
     return res
   }
+
+export const upsertBlobFactory =
```

Comment:
these are technically new, but only cause I had to split them out from a "service function" - I tried to change as little as possible in the actual implementation